awx/main/migrations/0145_auto_20210609_1652.py

```diff
@@ -0,0 +1,63 @@
+# Generated by Django 2.2.16 on 2021-06-09 16:52
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0144_event_partitions'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='credential',
+            old_name='managed_by_tower',
+            new_name='managed_by_controller',
+        ),
+        migrations.RenameField(
+            model_name='credentialtype',
+            old_name='managed_by_tower',
+            new_name='managed_by_controller',
+        ),
+        migrations.AlterField(
+            model_name='inventorysource',
+            name='source',
+            field=models.CharField(
+                choices=[
+                    ('file', 'File, Directory or Script'),
+                    ('scm', 'Sourced from a Project'),
+                    ('ec2', 'Amazon EC2'),
+                    ('gce', 'Google Compute Engine'),
+                    ('azure_rm', 'Microsoft Azure Resource Manager'),
+                    ('vmware', 'VMware vCenter'),
+                    ('satellite6', 'Red Hat Satellite 6'),
+                    ('openstack', 'OpenStack'),
+                    ('rhv', 'Red Hat Virtualization'),
+                    ('controller', 'Ansible Controller'),
+                ],
+                default=None,
+                max_length=32,
+            ),
+        ),
+        migrations.AlterField(
+            model_name='inventoryupdate',
+            name='source',
+            field=models.CharField(
+                choices=[
+                    ('file', 'File, Directory or Script'),
+                    ('scm', 'Sourced from a Project'),
+                    ('ec2', 'Amazon EC2'),
+                    ('gce', 'Google Compute Engine'),
+                    ('azure_rm', 'Microsoft Azure Resource Manager'),
+                    ('vmware', 'VMware vCenter'),
+                    ('satellite6', 'Red Hat Satellite 6'),
+                    ('openstack', 'OpenStack'),
+                    ('rhv', 'Red Hat Virtualization'),
+                    ('controller', 'Ansible Controller'),
```

Comment:
Same here - this should be Red Hat Ansible Automation Platform, and 'controller' has downstream implications.

Reply:
> and 'controller' has downstream implications.

Would "awx" have been a better choice? Myself, I think so, yes.

But very soon, users are going to be very frustrated with knowing what to use between tower / awx / controller.

Making any change is going to bring pain. But changing some things but not others, or worse, changing inconsistently, will bring even greater pain. There were valid arguments about the messaging, but I think that discussion has closed because we've already merged other user-facing stuff that has made a selection. To use another selection now would be bad.

Reply:
There's two questions:
- do we change the API namespace/setting from `tower`
- if so, what to

Assuming we *do* change, it, the standard for inventory and credential types is to use product names - satellite6, rhv, openstack, etc. So the change would be to 'aap'.